### PR TITLE
Enable tests in dashboard package

### PR DESCRIPTION
**Pull request by samiyaakhtar**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Enabling tests in dashboard package, used in recommendations

### CODE REVIEW

File: packages/dashboard/package.json [hunk 1]
```diff
@@ -29,9 +29,14 @@
     "update-snapshots": "node ../../scripts/build.js jest -u"
   },
   "devDependencies": {
+    "@types/enzyme": "3.1.5",
```
Comment (dzearing) [inline packages/dashboard/package.json]: When you change dependencies, re-run `npm run generate` from the root to regenerate the shrinkwrap.

Comment (dzearing) [reply packages/dashboard/package.json]: Also try to keep the versions consistent. (e.g. use whatever version of enzyme is used elsewhere, or, update them all.) reduces overhead and potential issues resulting from mismatched versions.

Comment (samiyaakhtar) [reply packages/dashboard/package.json]: I made sure all the versions are matching with other packages :)

Comment (cliffkoh) [reply packages/dashboard/package.json]: Have you ran rush generate?

Comment (samiyaakhtar) [reply packages/dashboard/package.json]: Just did, will merge now. Thanks!

**Pull request merged**